app/controllers/requests_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RequestsController < ApplicationController
+  def create
```

Comment:
I like this solution it makes a lot of sense and is easier on our end. since we're not storing that much data server side it seems fine to do it this way